uni/lib/l10n/intl_en.arb

```diff
@@ -294,6 +294,8 @@
   "@terms": {},
   "title": "Title",
   "@title": {},
+  "uc_info": "Open UC page in app",
```

Comment:
Probably just "Open UC page" is fine. Wdyt @bdmendes?

Reply:
Agreed